comms/discovery/rpcz/chat_blast_test.go

```diff
@@ -409,6 +409,183 @@ func TestChatBlast(t *testing.T) {
 		assert.Len(t, messages, 3)
 	}
 
+	// -------------------------------- tipper audience
+
+	// create some follower audiuence
+	_, err = tx.Exec(`
+	insert into user_tips
+		(slot, signature, sender_user_id, receiver_user_id, amount, created_at, updated_at)
+	values
+		(1, '', 201, 69, 2, $1, $1)
+	`, t0)
+	assert.NoError(t, err)
+
+	// 69 sends blast to supporters
+	_, err = chatBlast(tx, 69, t1, schema.ChatBlastRPCParams{
+		BlastID:  "blast_tippers_1",
+		Audience: schema.TipperAudience,
+		Message:  "thanks for your support",
+	})
+	assert.NoError(t, err)
+
+	// 201 should have a pending blast
+	{
+		pending, err := queries.GetNewBlasts(tx, ctx, queries.ChatMembershipParams{
+			UserID: 201,
+		})
+		assert.NoError(t, err)
+		assert.Len(t, pending, 1)
+	}
+
+	// 69 upgrades
```

Comment:
curious why 69 upgrades. should work either way but we should also have a test (maybe for another audience) where the recipient upgrades.